src/es6.js

```diff
@@ -7,50 +7,51 @@
 //----------------
 // const, =>, default parameters, arrow functions default return statements using ()
 
-var food = 'pineapple';
+const food = 'pineapple';
 
-var isMyFavoriteFood = function(food) {
+const isMyFavoriteFood = (food) =>{
   food = food || 'thousand-year-old egg'; //This sets a default value if `food` is falsey
   return food === 'thousand-year-old egg';
 };
 
-var isThisMyFavorite = isMyFavoriteFood(food);
+const isThisMyFavorite = isMyFavoriteFood(food);
 
 //----------------
 //const, class, template literals, enhanced object literals (foo: foo, -> foo,)
 
-var User = function(options) {
-  this.username = options.username;
-  this.password = options.password;
-  this.sayHi = function() {
-    return this.username + ' says hello!';
-  };
-}
+class User {
+  constructor(options) {
+  this.username = options.username
+  this.password = options.password
+  }
+  sayHi () {
+    return `${username} says hello!`;
+};
```

Comment:
A class also does not need a `;` at the end.  You are also missing a closing `}` after `sayHi`